cfme/tests/webui/test_general_ui.py

```diff
@@ -1,20 +1,38 @@
 import fauxfactory
 import pytest
+from widgetastic.utils import partial_match
 
 from cfme import test_requirements
+from cfme.common.host_views import ProviderAllHostsView
 from cfme.common.provider_views import CloudProviderAddView
 from cfme.common.provider_views import ContainerProviderAddView
 from cfme.common.provider_views import InfraProviderAddView
 from cfme.common.provider_views import InfraProvidersView
 from cfme.common.provider_views import PhysicalProviderAddView
+from cfme.common.provider_views import ProviderDetailsView
+from cfme.common.provider_views import ProviderTimelinesView
+from cfme.infrastructure.datastore import ProviderAllDatastoresView
+from cfme.infrastructure.provider import ProviderClustersView
+from cfme.infrastructure.provider import ProviderTemplatesView
+from cfme.infrastructure.provider import ProviderVmsView
 from cfme.infrastructure.provider.virtualcenter import VMwareProvider
 from cfme.markers.env_markers.provider import ONE_PER_TYPE
 from cfme.utils.appliance.implementations.ui import navigate_to
 from cfme.utils.wait import wait_for
 
-
 pytestmark = [test_requirements.general_ui]
 
+ALL_OPTIONS = {
+    "properties": {"Summary": ProviderDetailsView, "Timelines": ProviderTimelinesView},
+    "relationships": {
+        "Clusters (": ProviderClustersView,
```

Comment:
`(` has been added at the end of every key as a workaround for [BZ 1752372](https://bugzilla.redhat.com/show_bug.cgi?id=1752372).

Reply:
@valaparthvi the bz linked was closed as `NOTABUG`; how is the `(` relevant to the BZ?

Reply:
@john-dupuy that bug was not always reproducible, so to be on the safe side I added the `(`.